Database/Repositories/Implementations/MatchRatingStatsRepository.cs

```diff
@@ -23,22 +25,18 @@ public async Task<IEnumerable<IEnumerable<RatingAdjustment>>> GetForPlayerAsync(
         dateMax ??= DateTime.MaxValue;
 
         return await _context.RatingAdjustments
-            .Where(x =>
-                x.PlayerId == playerId
-                && x.AdjustmentType == RatingAdjustmentType.Match
-                && x.Match!.Tournament.Ruleset == ruleset
-                && x.Match.StartTime >= dateMin
-                && x.Match.StartTime <= dateMax
+            .Where(ra =>
+                ra.PlayerId == playerId
+                && ra.AdjustmentType == RatingAdjustmentType.Match
+                && ra.Ruleset == ruleset
+                && ra.Timestamp >= dateMin
+                && ra.Timestamp <= dateMax
             )
-            .Include(x => x.Match)
-            .ThenInclude(x => x!.Tournament)
-            .GroupBy(x => x.Match!.StartTime.Date)
+            .Include(ra => ra.Match!.Tournament)
+            .GroupBy(ra => ra.Timestamp)
```

Comment:
Is it necessary to group these before returning them? Last time I checked the only place where this is used flattens the list before using the contents anyways /shrug

Reply:
Probably makes more sense to let the caller of this function group them how they want to either way instead of this

Reply:
Agreed the repository shouldn't do the grouping here.